resource-managers/kubernetes/docker/src/main/dockerfiles/spark/bindings/python/Dockerfile

```diff
@@ -18,6 +18,10 @@
 ARG base_img
 
 FROM ${base_img:-spark}
+LABEL org.opencontainers.image.authors="Apache Spark project <dev@spark.apache.org>"
+LABEL org.opencontainers.image.licenses="Apache-2.0"
+LABEL org.opencontainers.image.ref.name="Apache Spark Python Image"
```

Comment:
PySpark?

Reply:
Ya, I also considered `Apache Spark PySpark Image` and `Apache Spark SparkR Image`.

But, I believe `Python` would be better because we have `Apache Spark Scala/Java Image` as a base image.

In addition, I'm not sure, but `Apache Spark Go Image` comes later. So, for consistency, language name could work extensively.

Reply:
Also, please see our docker hub. The Docker convention in the name is languages like `scala`, `r`, `python3` instead of `SparkR` or `PySpark`.

![Screenshot 2024-08-14 at 23 33 02](https://github.com/user-attachments/assets/a879d620-7f68-4a08-a201-830f881783ea)

Reply:
I was considering whether it will conflict with the https://pypi.org/project/pyspark-connect/ or not.

But it looks fine to me now